### PR TITLE
regex fixup for goodmorning and goodnight

### LINKED ISSUE #79: Improve goodnight and goodmorning regex
regex bad plz fix

_Issue created by Ginger [here](https://chat.stackexchange.com/transcript/106764?m=63732895#63732895)_

### PR DESCRIPTION
**Pull request by PlaceReporter99**
fixes #79 i think

### CODE REVIEW

File: vyxalbot2/commands.py [hunk 1]
```diff
@@ -53,8 +53,8 @@
     (
         r"(much |very |super |ultra |extremely )*(good|great|excellent|gaming) bot!*",
     ): "!good-bot",
-    (r"(hello|howdy|mornin['g]|evenin['g])( y'?all)?",): "hello",
-    (r"((good)?bye|see ya\!?|'night|goodnight)( y'?all)?",): "goodbye",
+    (r"(hello|howdy|(good )?mornin['g]|(good )?evenin['g])( y'?all)?",): "hello",
+    (r"((good)?bye|see ya\!?|'night|(good|night )night)( y'?all)?",): "goodbye",
```
Comment (ysthakur) [inline vyxalbot2/commands.py:57]:
```suggestion
    (r"((good)?bye|see ya\!?|'night|(good ?|night )?night)( y'?all)?",): "goodbye",
```
To allow `good night` with a space in between, as well as just `night`

**Review by ysthakur — APPROVED** [review]
LGTM but probably wait for Ginger to merge

**Comment by ysthakur** [comment]
Wait, you need to merge this into `dev`, not `main`. @GingerIndustries could you make it so that `dev` is the default branch so people don't get confused?

**Comment by ysthakur** [comment]
Okay, I don't know what happened

**Comment by GingerIndustries** [comment]
I keep intending to just nuke `dev` and only use `main`

**Comment by ysthakur** [comment]
Sorry, didn't realize that would mess it up even more. Not sure how to fix it

**Comment by PlaceReporter99** [comment]
Let me close this and open a new one @ysthakur @GingerIndustries

**Comment by PlaceReporter99** [comment]
Go to PR #95

**Pull request closed without merging**